src/PortToDocs/src/libraries/Configuration.cs

```diff
@@ -68,7 +68,6 @@ private enum Mode
 
         public readonly string BinLogPath = "output.binlog";
         public bool BinLogger { get; private set; } = false;
-        public FileInfo? CsProj { get; set; }
```

Comment:
This was a remnant from when the two projects were a single one. `CsProj` only applies to `ToTripleSlashPorter`, so I'm removing it from here.